packages/playground/src/samples/allOf.js

```diff
@@ -5,15 +5,15 @@ module.exports = {
       {
         properties: {
           lorem: {
-            type: ["string", "number"],
+            type: ["string", "boolean"],
+            default: true,
           },
         },
       },
       {
         properties: {
           lorem: {
             type: "boolean",
-            minLength: 5,
```

Comment:
@epicfaace 

Like the addition of being able to select the bootstrap theme in the playground :P

with regards to my PR https://github.com/rjsf-team/react-jsonschema-form/pull/1610 and the allof, and my comments regarding merging of schema's

to quote

> To validate against allOf, the given data must be valid against all of the given subschemas.

> Note that it’s quite easy to create schemas that are logical impossibilities with allOf. The following example creates a schema that won’t validate against anything (since something may not be both a string and a number at the same time):

```
{
  "allOf": [
    { "type": "string" },
    { "type": "number" }
  ]
}
"No way"
```

https://json-schema.org/understanding-json-schema/reference/combining.html

here the type to validate against should be both a string and a boolean, this would essentially mean that we could never validate against the lorem field here as the entry could be both but i believe the merge will make it so that it picks the common one.

Reply:
Yeah, exactly, so the schema in this file will, once merged, have `"lorem": {"type": "boolean", "default": true}`.

Reply:
which I believe to be incorrect behavior